lesson11/main.js

```diff
@@ -22,18 +21,19 @@ async function fetchData() {
       console.error('エラー');
    } finally {
       console.log('処理が終了しました。')
-      endLoading();
    }
 }
 
-async function runCreateListElement() {
+async function init() {
+   startLoading();
    try {
       const result = await fetchData();
       createListElements(result);
-   } catch {
+   } catch(e) {
       alert(`データが取得できませんでした`)
```

Comment:
Sorry, I missed it.
```
alert(`データが取得できませんでした`)
```

Here we use single or double quotation marks instead of back quotation marks because it is just a string.

Reply:
Thanks, I also mistaked. I fixed there you pointed out.

652338e